postgres/lib/dwconnect.js

```diff
@@ -21,22 +21,23 @@ module.exports = function(config, columnConfig) {
 	client.getDimensionColumn = columnConfig.dimColumnTransform;
 
 	client.importFact = function(stream, table, ids, callback) {
+		const stagingTbl = `stage.staging_${table}`;
```

Comment:
`stage.staging_${table}` will only work if the database has a stage schema.
this needs to be something like `${stageSchema}.staging_${table}` where the default value of stageSchema is "public"